signal_scanner_bot/twitter.py

```diff
@@ -35,16 +36,73 @@ def get_api():
 ################################################################################
 def send_tweet(tweet: str, timestamp: datetime, api: tweepy.API) -> None:
     hashtags = " ".join(SEND_HASHTAGS)
-    if len(tweet + hashtags) >= 260:
-        # TODO: better
-        log.warning(f"Cannot tweet message, exceeds length: {tweet}")
-        return
-
-    formatted = dedent(
-        f"""
-    {tweet} @ {timestamp.strftime('%l:%M:%S%p').strip()}
-
-    {hashtags}
-    """
-    )
-    api.update_status(formatted)
+
+    # Check if tweet is longer than 280 minus a 20 char padding, add to list if
+    # not, continue with dividing up text if so
+    if len(tweet + hashtags) >= TWEET_MAX_SIZE - TWEET_PADDING:
```

Comment:
I think it might be a good idea to move this logic out into a separate function that's present in this module. We could have a `send_single_tweet` function that still only ever sends one tweet, and then a `send_multiple_tweets` function that splits up the tweets and sends them out (or just receives a list of split up tweets). Then we can have a higher level `send_tweet` that accepts the signal message and dispatches to the appropriate method depending on message length. Thoughts?